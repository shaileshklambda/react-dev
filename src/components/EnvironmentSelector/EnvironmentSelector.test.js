import React from "react";
import renderer from "react-test-renderer";
import Environmentselector from ".";

it("Environmentselector: default", () => {
  const component = renderer.create(<Environmentselector />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
