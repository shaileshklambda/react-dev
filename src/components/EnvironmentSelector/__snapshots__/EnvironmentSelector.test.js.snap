// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Environmentselector: default 1`] = `
<div>
  <div
    className="testContainer panel panel-default"
  >
    <div
      className="panel-body"
    >
      <div
        className="row"
      >
        <div
          className="col-md-1"
        >
          <span />
        </div>
        <div
          className="col-md-4"
        >
          <span
            className="envHeading"
          >
            Environment
          </span>
        </div>
      </div>
      <div
        className="row row"
      >
        <div
          className="col-md-1"
        >
          <img
            className=""
            src="computer.svg"
          />
        </div>
        <div
          className="col-md-5"
        >
          <div
            className="css-10nd86i selectClass"
            onKeyDown={[Function]}
          >
            <div
              className="css-vj8t7z"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className="css-1hwfws3"
              >
                <div
                  className="css-1492t68"
                >
                  Select A Browser
                </div>
                <div
                  className="css-1g6gooi"
                >
                  <div
                    className=""
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-2-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      theme={
                        Object {
                          "borderRadius": 4,
                          "colors": Object {
                            "danger": "#DE350B",
                            "dangerLight": "#FFBDAD",
                            "neutral0": "hsl(0, 0%, 100%)",
                            "neutral10": "hsl(0, 0%, 90%)",
                            "neutral20": "hsl(0, 0%, 80%)",
                            "neutral30": "hsl(0, 0%, 70%)",
                            "neutral40": "hsl(0, 0%, 60%)",
                            "neutral5": "hsl(0, 0%, 95%)",
                            "neutral50": "hsl(0, 0%, 50%)",
                            "neutral60": "hsl(0, 0%, 40%)",
                            "neutral70": "hsl(0, 0%, 30%)",
                            "neutral80": "hsl(0, 0%, 20%)",
                            "neutral90": "hsl(0, 0%, 10%)",
                            "primary": "#2684FF",
                            "primary25": "#DEEBFF",
                            "primary50": "#B2D4FF",
                            "primary75": "#4C9AFF",
                          },
                          "spacing": Object {
                            "baseUnit": 4,
                            "controlHeight": 38,
                            "menuGutter": 8,
                          },
                        }
                      }
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="css-1wy0on6"
              >
                <span
                  className="css-d8oujb"
                />
                <div
                  aria-hidden="true"
                  className="css-1ep9fjw"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="css-19bqh2r"
                    focusable="false"
                    height={20}
                    viewBox="0 0 20 20"
                    width={20}
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-md-5"
        >
          <div
            className="css-10nd86i selectClass"
            onKeyDown={[Function]}
          >
            <div
              className="css-vj8t7z"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className="css-1hwfws3"
              >
                <div
                  className="css-1492t68"
                >
                  Select An OS
                </div>
                <div
                  className="css-1g6gooi"
                >
                  <div
                    className=""
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-3-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      theme={
                        Object {
                          "borderRadius": 4,
                          "colors": Object {
                            "danger": "#DE350B",
                            "dangerLight": "#FFBDAD",
                            "neutral0": "hsl(0, 0%, 100%)",
                            "neutral10": "hsl(0, 0%, 90%)",
                            "neutral20": "hsl(0, 0%, 80%)",
                            "neutral30": "hsl(0, 0%, 70%)",
                            "neutral40": "hsl(0, 0%, 60%)",
                            "neutral5": "hsl(0, 0%, 95%)",
                            "neutral50": "hsl(0, 0%, 50%)",
                            "neutral60": "hsl(0, 0%, 40%)",
                            "neutral70": "hsl(0, 0%, 30%)",
                            "neutral80": "hsl(0, 0%, 20%)",
                            "neutral90": "hsl(0, 0%, 10%)",
                            "primary": "#2684FF",
                            "primary25": "#DEEBFF",
                            "primary50": "#B2D4FF",
                            "primary75": "#4C9AFF",
                          },
                          "spacing": Object {
                            "baseUnit": 4,
                            "controlHeight": 38,
                            "menuGutter": 8,
                          },
                        }
                      }
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="css-1wy0on6"
              >
                <span
                  className="css-d8oujb"
                />
                <div
                  aria-hidden="true"
                  className="css-1ep9fjw"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="css-19bqh2r"
                    focusable="false"
                    height={20}
                    viewBox="0 0 20 20"
                    width={20}
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row row"
      >
        <div
          className="col-md-1"
        >
          <img
            className="phoneEnvIcon"
            src="phone_2.svg"
          />
        </div>
        <div
          className="col-md-5"
        >
          <div
            className="css-10nd86i selectClass"
            onKeyDown={[Function]}
          >
            <div
              className="css-vj8t7z"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className="css-1hwfws3"
              >
                <div
                  className="css-1492t68"
                >
                  Select A Mobile Browser
                </div>
                <div
                  className="css-1g6gooi"
                >
                  <div
                    className=""
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-4-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      theme={
                        Object {
                          "borderRadius": 4,
                          "colors": Object {
                            "danger": "#DE350B",
                            "dangerLight": "#FFBDAD",
                            "neutral0": "hsl(0, 0%, 100%)",
                            "neutral10": "hsl(0, 0%, 90%)",
                            "neutral20": "hsl(0, 0%, 80%)",
                            "neutral30": "hsl(0, 0%, 70%)",
                            "neutral40": "hsl(0, 0%, 60%)",
                            "neutral5": "hsl(0, 0%, 95%)",
                            "neutral50": "hsl(0, 0%, 50%)",
                            "neutral60": "hsl(0, 0%, 40%)",
                            "neutral70": "hsl(0, 0%, 30%)",
                            "neutral80": "hsl(0, 0%, 20%)",
                            "neutral90": "hsl(0, 0%, 10%)",
                            "primary": "#2684FF",
                            "primary25": "#DEEBFF",
                            "primary50": "#B2D4FF",
                            "primary75": "#4C9AFF",
                          },
                          "spacing": Object {
                            "baseUnit": 4,
                            "controlHeight": 38,
                            "menuGutter": 8,
                          },
                        }
                      }
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="css-1wy0on6"
              >
                <span
                  className="css-d8oujb"
                />
                <div
                  aria-hidden="true"
                  className="css-1ep9fjw"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="css-19bqh2r"
                    focusable="false"
                    height={20}
                    viewBox="0 0 20 20"
                    width={20}
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-md-5"
        >
          <div
            className="css-10nd86i selectClass"
            onKeyDown={[Function]}
          >
            <div
              className="css-vj8t7z"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className="css-1hwfws3"
              >
                <div
                  className="css-1492t68"
                >
                  Select A Mobile Platform
                </div>
                <div
                  className="css-1g6gooi"
                >
                  <div
                    className=""
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-5-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      theme={
                        Object {
                          "borderRadius": 4,
                          "colors": Object {
                            "danger": "#DE350B",
                            "dangerLight": "#FFBDAD",
                            "neutral0": "hsl(0, 0%, 100%)",
                            "neutral10": "hsl(0, 0%, 90%)",
                            "neutral20": "hsl(0, 0%, 80%)",
                            "neutral30": "hsl(0, 0%, 70%)",
                            "neutral40": "hsl(0, 0%, 60%)",
                            "neutral5": "hsl(0, 0%, 95%)",
                            "neutral50": "hsl(0, 0%, 50%)",
                            "neutral60": "hsl(0, 0%, 40%)",
                            "neutral70": "hsl(0, 0%, 30%)",
                            "neutral80": "hsl(0, 0%, 20%)",
                            "neutral90": "hsl(0, 0%, 10%)",
                            "primary": "#2684FF",
                            "primary25": "#DEEBFF",
                            "primary50": "#B2D4FF",
                            "primary75": "#4C9AFF",
                          },
                          "spacing": Object {
                            "baseUnit": 4,
                            "controlHeight": 38,
                            "menuGutter": 8,
                          },
                        }
                      }
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="css-1wy0on6"
              >
                <span
                  className="css-d8oujb"
                />
                <div
                  aria-hidden="true"
                  className="css-1ep9fjw"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="css-19bqh2r"
                    focusable="false"
                    height={20}
                    viewBox="0 0 20 20"
                    width={20}
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
